resources/keywords/dbaas_policy.resource

```diff
@@ -100,16 +103,91 @@ Click If Element Present
+    END
+    IF    ${label}
+        Click Element    ${btnToggleConnNSSelect}
+        Element Should Be Visible    ${btnToggleMatchExp}
+        Click Element    ${btnToggleMatchExp}
+        Element Should Be Visible    ${btnAddMatchExp}
+        Add Policy Connection Nsselector    ${addLabel}
+    END
+
+User Creates DBaaSPolicy With ConnectionNamespace For ${namespace} As ${DBaaSPersona}
+    Set ConnectionNamespace For DBaaSPolicy
+    User Creates DBaaSPolicy For ${namespace} From Admin View As ${DBaaSPersona}
+
+Set ConnectionNamespace For DBaaSPolicy
+    Set Test Variable   ${nsAndLabel}   namespace
+    Set Test Variable   @{addNS}    ${connectionNS}
+
+DBaaSInventory Is Available For ${targetns}
+    User Filters Project ${targetns} On Project DropDown And Navigates To Database Access Page
+    Imported Provider Account Available On Database Access Screen
+    User Navigates To Add ${DBProvider} To Topology Screen
+    Select Given Provider Account On Add Topology Screen
+
+User Creates DBaaSPolicy As ${user} With ConnectionNamespace And Imports ${databaseProvider} Provider Account
+    The User Logs In To Openshift As ${user}
+    User Creates DBaaSPolicy With ConnectionNamespace For ${DBaaSTestNS} As ${user}
+    User Is Able To Import MongoDB Provider Account For ${DBaaSTestNS}
+
+User Creates DBaaSPolicy As ${user} Without ConnectionNamespace And Imports ${databaseProvider} Provider Account
+    The User Logs In To Openshift As ${user}
+    User Creates DBaaSPolicy For ${DBaaSTestNS} From Admin View As ${user}
+    User Is Able To Import MongoDB Provider Account For ${DBaaSTestNS}
+
+DBaaSInventory Is Not Available For ${targetns}
+    User Navigates To Add ${DBProvider} To Topology Screen
+    Wait Until Page Contains Element    ${txtNoDBInstance}    timeout=15s
+
+User Creates ${namespace} And Aligns ${access} Role To The ${user}
+    Set Test Variable    ${access}
+    Set Test Variable    ${user}
+    Login To OpenShift CLI As Cluster Admin
+    Check LDAP Configuration For The Cluster
+    Add DBaaS Namespace     ${namespace}
+    User Aligns ${access} Role To ${namespace} For ${user}
+    Close Browser
+    The User Logs In To Openshift As ${user}
+
+User Aligns ${access} Role To ${namespace} For ${user}
+    Login To OpenShift CLI As Cluster Admin
+    ${cmd} =    Set Variable    oc adm policy add-role-to-user ${access} ${${user}.USERNAME} -n ${namespace}
+    Execute ${cmd} And Raise Exception For Error
+
+The Namespace ${namespace} Is Not Available On The Project Dropdown
+    User Navigates To Installed Operators Under Operators
+    Wait Until Page Contains Element    ${drpDwnProjectSelect}    timeout=15s
+    Click Element    ${drpDwnProjectSelect}
+    Element Should Be Visible    ${txtBxProjectSelect}
+    Input Text    ${txtBxProjectSelect}    ${namespace}
+    ${btnProjectSelect}=    Replace String    ${btnProjectSelect}    <<project>>    ${namespace}
+    Element Should Not Be Visible    ${btnProjectSelect}
```

Comment:
Add new line in the end of this file

Reply:
I am surprised as the file contains the new line at the end of this file and it is not showing up on github. Even I dont see a warning message for this. Something to learn, but I dont think this would be blocker for this PR. Please let me know.

Reply:
Maybe it relates to robotidy?

Reply:
I dont think so

Reply:
OK strange...